bootstrap4/templatetags/bootstrap4.py

```diff
@@ -214,52 +239,59 @@ def bootstrap_css():
 
 
 @register.simple_tag
-def bootstrap_jquery():
+def bootstrap_jquery(jquery='full'):
     """
     Return HTML for jQuery tag.
 
-    Adjust url in settings. If no url is returned, we don't want this
-    statement to return any HTML.
+    Adjust the url dict in settings. If no url is returned, we
+    don't want this statement to return any HTML.
     This is intended behavior.
 
-    Default value: ``None``
+    Default value: ``full``
```

Comment:
It seems you interpret this as the default value for the first argument and that the docstring was quite out of sync with the current functionality. I'd say this line can be removed entirely, as the default value is documented under **Parameters**.

Reply:
You're probably correct, I was just following the pattern / artifact from Bootstrap 3. I'll get that in place today.